packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -325,88 +299,131 @@ export default function AccessRequirementList(
-          />
-        )
-        break
-      case RequestDataStep.PROMPT_LOGIN:
-        dialogTitle = 'Please Log In'
-        renderContent = (
-          <Box className={'AccessRequirementList login-modal '}>
+  let renderContent = <></>
+  switch (requestDataStep) {
+    case RequestDataStep.UPDATE_RESEARCH_PROJECT:
+      renderContent = (
+        <ResearchProjectForm
+          managedACTAccessRequirement={managedACTAccessRequirement!}
+          onSave={researchProject => {
+            requestDataStepCallback({
+              managedACTAccessRequirement,
+              step: RequestDataStep.UPDATE_ACCESSORS_AND_FILES,
+              researchProjectId: researchProject.id,
+            })
+          }}
+          onHide={onHide}
+        />
+      )
+      break
+    case RequestDataStep.UPDATE_ACCESSORS_AND_FILES:
+      renderContent = (
+        <DataAccessRequestAccessorsFilesForm
+          researchProjectId={researchProjectId}
+          managedACTAccessRequirement={managedACTAccessRequirement!}
+          subjectId={subjectId ?? ''}
+          subjectType={subjectType ?? RestrictableObjectType.ENTITY}
+          onHide={onHide}
+          onCancel={dataAccessRequestInProgress => {
+            requestDataStepCallback({
+              step: RequestDataStep.PROMPT_CANCEL,
+              dataAccessRequest: dataAccessRequestInProgress,
+            })
+          }}
+          onSubmissionCreated={() => {
+            requestDataStepCallback({ step: RequestDataStep.COMPLETE })
+          }}
+        />
+      )
+      break
+    case RequestDataStep.PROMPT_CANCEL:
+      renderContent = (
+        <CancelRequestDataAccess
+          modifiedDataAccessRequest={dataAccessRequest}
+          onHide={onHide} // for closing dialogs
+        />
+      )
+      break
+    case RequestDataStep.PROMPT_LOGIN:
+      dialogTitle = 'Please Log In'
+      renderContent = (
+        <>
+          <DialogBaseTitle title={dialogTitle} onCancel={onHide} />
+          <DialogContent className={'AccessRequirementList login-modal '}>
             <StandaloneLoginForm
               sessionCallback={() => {
                 window.location.reload()
               }}
             />
-          </Box>
-        )
-        break
-      case RequestDataStep.COMPLETE:
-        renderContent = <RequestDataAccessSuccess onHide={onHide} />
-        break
-      case RequestDataStep.SHOW_ALL_ARS:
-      default:
-        renderContent = content
-        dialogActions = (
-          <Button variant="contained" onClick={onHide}>
-            Close
-          </Button>
-        )
-    }
+          </DialogContent>
+        </>
+      )
+      break
+    case RequestDataStep.COMPLETE:
+      renderContent = <RequestDataAccessSuccess onHide={onHide} />
+      break
+    case RequestDataStep.SHOW_ALL_ARS:
+    default:
+      renderContent = (
+        <>
+          <DialogBaseTitle title={dialogTitle} onCancel={onHide} />
+          <DialogContent>
```

Comment:
Effectively revert the previous change by adding `DialogBaseTitle`, `DialogContent`, `DialogActions`